Archive/Imo/Imo2005Q3.lean

```diff
@@ -24,8 +24,6 @@ and then making use of `xyz ≥ 1` to show `(x^5-x^2)/(x^3*(x^2+y^2+z^2)) ≥ (x
 -/
 
 
-namespace imo2005_q3
```

Comment:
Why are you removing the namespace?

Reply:
fixed: I put them back after I realized, I just have to change the name imo2005_q3 -> Imo2005Q3